osquery/config/plugins/tls.cpp

```diff
@@ -46,32 +48,29 @@ CLI_FLAG(uint64,
          0,
          "Optional interval in seconds to re-read configuration");
 
+/// Option to always do enroll at startup time
+CLI_FLAG(bool,
+         config_tls_enroll_on_startup,
```

Comment:
Can we make this enroll_always and have the re-enroll expectation happen in all enroll plugins (there's only the TLS for now ha)?